scss/_settings_placeholders.scss

```diff
@@ -2,10 +2,20 @@
 @import 'settings_colors';
 
 // Global placeholder settings
-$input-border-thickness: 1.5px;
+:root {
+  // On low pixel density devices a 1.5px border would computed as 1px.
+  // If we use $input-border-thickness = 1.5px in the calculation and the user is on a low pixel density screen
+  // then the border is rendered as 1px, but we still eg. subtract 1.5px from the padding which will throw off the baseline grid alignment.
+  // So we have to check if the users screen is capable of displaying 0.5px steps at runtime. This is only possible with CSS variables.
+  --input-border-thickness: 1.5px;
```

Comment:
Small suggestion, to avoid any future name clashes we namespace our CSS variables with `--vf`, so in this case `--vf-input-border-thickness` (I guess maybe we can have a better name later when we properly define this as a token).